tests/NLog.UnitTests/LayoutRenderers/Log4JXmlTests.cs

```diff
@@ -156,6 +156,10 @@ public void Log4JXmlTest()
                                         break;
 #endif
 
+                                    case "nlogPropertyKey":
```

Comment:
Not sure I understand. It is part of the standard log4j-properties because of IncludeAllProperties=true. The non-standard nlog-properties can only be handled by custom parsers.